_posts/2020-07-19-stage-set-for-knockouts.md

```diff
@@ -28,7 +28,7 @@ for a number of other awards, including the Committee Award (given to the team
 that displays the most extraordinary ingenuity in their robot), and the Online
 Presence Award (for the team that is judged to have the best online presence).
 Full details of these awards and those for earlier stages of the competition are
-available in our [rulebook](/docs/resources/2020/rulebook.pdf)
+available in our [rulebook](https://studentrobotics.org/docs/docs/resources/2020/rulebook.pdf)
```

Comment:
Think there's an extra layer of `/docs` here

Reply:
eb1a36860a377ea49d4dd70757ef8a416cc76b4e